portal/src/views/projects/ProjectsView.vue

```diff
@@ -50,6 +50,9 @@ export default {
         }),
     },
     mounted() {
+        setTimeout(() => {
```

Comment:
I try and keep my debugging stuff out of commits.

Reply:
Ops, thanks for noticing this!